data/benefits/openfisca/gratuite_musees_monuments.yml

```diff
@@ -1,9 +1,17 @@
 label: accès gratuit aux musées et monuments nationaux
 institution: etat
-description: Les jeunes de moins de 26 ans habitant dans l'Union européenne peuvent accéder gratuitement à certains musées et monuments nationaux, sur simple présentation d'une pièce d'identité.
+description: Les jeunes de 18 à 25 ans qui résident dans un Etat membre de l'Union européenne ou faisant partie de l'Espace économique européen peuvent accéder gratuitement aux collections permanentes des musées nationaux et entrer gratuitement dans les monuments nationaux, sur simple présentation d'une pièce d'identité.
 conditions:
-  - Présenter au guichet du musée ou du monument concerné avec une pièce justifiant de son âge et de sa résidence. Cette pièce peut être une carte d'identité, un passeport ou une carte de séjour.
-link: https://www.service-public.fr/particuliers/vosdroits/F20348
+  - Présenter au guichet du musée ou du monument concerné une pièce justifiant de votre âge et de votre résidence. Cette pièce peut être une carte d'identité, un passeport ou une carte de séjour.
```

Comment:
J'ai eu un bug mental, concernant la phrase "une pièce justifiant de votre âge et de votre résidence", j'ai d'abord pensé à j'ai besoin de deux pièces justificatives (une d'identité et une de domicile) mais au final non grâce à la suite de ta phrase.

Reply:
Oui exactement : il n'y a pas de page officielle qui parle des deux dispositifs.
Si on décide de scinder en deux aides, il faut le faire dans OF.
Je pense que ce n'est pas absolument indispensable mais qu'en penses-tu toi ?